@uportal/esco-content-menu/src/services/flattenPortlets.js

```diff
@@ -0,0 +1,47 @@
+export default function flattenPortlets(elem) {
```

Comment:
Maybe this could leverage https://github.com/uPortal-contrib/uPortal-web-components/tree/master/%40uportal/portlet-registry-to-array ?

Reply:
done, but with some local change and so moved locally